File: simple_benchmarks/bmark1.jl
using UnROOT

const t_lz4 = LazyTree(ROOTFile("lz4_Run2012BC_DoubleMuParked_Muons.root"),"Events");
function foo(t)
    s = 0.0f0
    for evt in t
        s += sum(evt.Muon_pt)
    end
    s
end

@time foo(t_lz4)

File: simple_benchmarks/bmark1.py
import awkward1 as ak
import time
import uproot # uproot4

f = uproot.open("uncompressed_Run2012BC_DoubleMuParked_Muons.root")
t = f["Events"]

t0 = time.time()
print(ak.sum(t["Muon_pt"].array()))
t1 = time.time()
print(t1-t0)
